.circleci/config.yml

```diff
@@ -47,6 +77,87 @@ jobs:
       - run:
           name: Run Checks
           command: bin/ci-checks-js.sh
+  android-device-checks:
+    docker:
+    - image: circleci/android:api-27-node8-alpha
+    steps:
+      - checkout
+      - run:
+          name: Checkout Gutenberg
+          command: |
+            git submodule init
+            git submodule update --recursive
+      - yarn-install
+      - run:
+          name: Accept Licenses
+          command: |
+            yes | sdkmanager --licenses > /dev/null
+            sleep 10
+            sdkmanager --update
+          background: true
+      - run:
+          name: Set Environment Variables
+          command: |
+            echo 'export TEST_RN_PLATFORM=android' >> $BASH_ENV
+            echo 'export TEST_ENV=sauce' >> $BASH_ENV
+      - build-android
+      - run:
+          name: Upload apk to sauce labs
+          command: |
+            curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" -X POST -H "Content-Type: application/octet-stream" https://saucelabs.com/rest/v1/storage/automattic/Gutenberg.apk?overwrite=true --data-binary @./android/app/build/outputs/apk/debug/app-debug.apk
+      - run:
+          name: Run Device Tests
+          command: yarn device-tests
+  ios-device-checks:
+    macos:
+      xcode: "10.0"
+    steps:
+    - checkout
+    - run:
+        name: Checkout Gutenberg
+        command: |
+          git submodule init
+          git submodule update --recursive
+    - run: yarn clean:install
+    - run:
+        name: Set Environment Variables
+        command: |
+          echo 'export TEST_RN_PLATFORM=ios' >> $BASH_ENV
+          echo 'export TEST_ENV=sauce' >> $BASH_ENV
+    - restore_cache:
+        name: Restore Dependencies Cache
+        keys:
+        - dependencies-{{ checksum "react-native-aztec/ios/Cartfile.resolved" }}-{{
+          checksum "yarn.lock" }}
+        - dependencies-{{ checksum "react-native-aztec/ios/Cartfile.resolved" }}
+        - dependencies-
+    - run:
+        name: Yarn preios
+        command: yarn preios
+    - save_cache:
+        name: Save Dependencies Cache
+        key: dependencies-{{ checksum "react-native-aztec/ios/Cartfile.resolved" }}-{{
+          checksum "yarn.lock" }}
+        paths:
+        - react-native-aztec/ios/Carthage
+        - ~/.rncache
+    - build-ios
+    - run:
+        name: Zip up .app file
+        command: |
+          cp ./ios/main.jsbundle ./ios/build/Gutenberg/Build/Products/Release-iphonesimulator/gutenberg.app/main.jsbundle
+          cd ./ios/build/Gutenberg/Build/Products/Release-iphonesimulator/
+          zip -r ./Gutenberg.app.zip ./gutenberg.app
+          mv ./Gutenberg.app.zip ../../../../../../Gutenberg.app.zip
+          cd ../../../../../../
```

Comment:
This has no effect since the next command always starts with a new shell in `$CIRCLE_WORKING_DIRECTORY`

Reply:
🤦🏽‍♂️Yeah you're right I'll change this